scripts/common/rook.sh

```diff
@@ -323,14 +291,13 @@ function rook_10_to_14() {
     # todo store these files ourselves
     curl -sSL https://raw.githubusercontent.com/rook/rook/v1.1.9/cluster/examples/kubernetes/ceph/upgrade-from-v1.0-crds.yaml | kubectl apply -f -
 
-    # todo upgrade ceph image to 14.2.5 here
     echo "Upgrading ceph to v14.2.5"
     kubectl -n rook-ceph patch CephCluster rook-ceph --type=merge -p '{"spec": {"cephVersion": {"image": "ceph/ceph:v14.2.5-20191210"}}}'
-    spinner_until -1 is_ceph_rollout_complete "14.2.5"
+    $DIR/bin/kurl rook wait-for-ceph-version "14.2.5"
```

Comment:
when something goes wrong with upgrades it is most often the case that scripts fail waiting for ceph to roll out with a timeout of 10 minutes. should we do something similar here or have the user manually kill the script? i do not have a strong opinion either way.

Reply:
I can certainly see putting in a time limit, but I wasn't planning to until I had seen an issue.